assets/prometheus-k8s/rules.yaml

```diff
@@ -591,6 +591,7 @@ spec:
           occur. Check the OpenShift documentation to learn how to configure notifications
           with Alertmanager.
       expr: cluster:alertmanager_routing_enabled:max == 0
+      for: 30m
```

Comment:
Lets then adjust to 15m and that should ™️ give enough time to fire watchdog alert after upgrades. I would say we can also add note in bugzilla for QE to verify that when cluster first spins up and user logs in this alert is firing. WDYT?